tests/scripts/thread-cert/test_detach.py

```diff
@@ -145,10 +145,10 @@ def test(self):
         self.assertEqual(leader.get_state(), 'disabled')
 
         leader.start()
-        self.simulator.go(config.LEADER_STARTUP_DELAY)
+        self.simulator.go(config.LEADER_RESET_DELAY)
```

Comment:
add check it's detached when T < LEADER_STARTUP_DELAY?

Reply:
Done